### ISSUE TITLE
Warn when a package being installed is x months old

### ISSUE THREAD

**Issue by ferventcoder** [chocolatey/choco#1156]
When a package is over a certain number of months old, Chocolatey should warn a user about installing it. This will give them a quick 10-15 second opportunity to decide not to move forward if auto confirm is on, and will wait for their confirmation otherwise.

**Comment by AdmiringWorm** [comment]
I've seen this a few times now, and finally decided to voice my opinion on this (based on how I see choco as, namely a package manager).

With that said, I'm against such a feature (unless implemented as specified below).
Installing a package is IMO is the responsibility of the user to decide that it is safe enough to run on their system, there could be a number of reasons why a package haven't been updated (no new package have been released, maybe because there haven't been a need for it (first thought)).

With that said, there are few scenarios I can see it useful though:
1. A new package version have been released and is currently in moderation queue
2. A new package version have been released, but is currently in rejected state
3. The user is installing an older version of the package.
4. A new software release is known to be released, but the package have not yet been updated for that version *(Not possible though, as far as I know)*.


I see that implementing such a feature (although could be useful) would suggest that choco is moving away from being a package manager to something else (I don't know what though)

**Comment by ferventcoder** [comment]
To be fair - working with native installers already suggests that choco is more than just a package manager. Portables and runtime software in the package are "true" packages, but let's be honest, if choco was restricted to just that it would not have garnered the popularity it currently has.

It trends more into software management, which encapsulates package management in addition to other features. That said we consider FOSS more towards package mgmt and the biz edition complete software managment.

**Comment by AdmiringWorm** [comment]
> To be fair - working with native installers already suggests that choco is more than just a package manager.

True, never though about it that way. Mostly because of how the windows ecosystem is.
